scenes/enemies/Sheep/Sheep.gd

```diff
@@ -21,13 +21,14 @@ func _init():
 
 
 func _ready():
+	# Make sure to connect to all signals before super is called
```

Comment:
Oh interesting, what was the issue that was happening as a result of not doing this?

Reply:
When you need to listen to the result of a rpc, you should always first subscribe to the response before calling the actual the rpc. The rpc has been moved to the super's ready (in JEnemyBody2D), thus the subscribe should be done before this function.